types/ahocorasick/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// Type definitions for ahocorasick 1.0.2
+// Project: https://github.com/BrunoRB/ahocorasick
+
+export default class AhoCorasick {
+    constructor(keywords: string[]);
+
+    search(match: string): object[]
```

Comment:
A more accurate type can be used for this part.

```suggestion
    search(match: string): [number, string][]
```